src/viur/core/bones/key.py

```diff
@@ -47,8 +47,12 @@ def singleValueFromClient(self, value, skel, bone_name, client_data):
                 return self.getEmptyValue(), [ReadFromClientError(ReadFromClientErrorSeverity.Invalid, e.args[0])]
         else:
             try:
-                key = db.normalizeKey(db.Key.from_legacy_urlsafe(value))
-            except:
+                if isinstance(value, str):
+                    key = db.normalizeKey(db.Key.from_legacy_urlsafe(value))
+                elif isinstance(value, db.Key):
+                    key = db.normalizeKey(value)
```

Comment:
What happens here when key is either not `isinstance(value, str)` and not `isinstance(value, db.Key)`? In this case, key will left as is, and program flow continues.

I would like suggest:
```suggestion
                if isinstance(value, db.Key):
                    key = db.normalizeKey(value)
                else:
                    key = db.normalizeKey(db.Key.from_legacy_urlsafe(value))
```
to keep closer to the previous behavior.